File: Prog Multimedia/2ºTrimestre/Layouts Inflates/LayoutInflater/app/src/main/java/com/example/layoutinflater/inflate4/MainActivity4.java
package com.example.layoutinflater.inflate4;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.LinearLayout;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.example.layoutinflater.R;

public class MainActivity4 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // Inflar el diseño XML
        View rootView = LayoutInflater.from(this).inflate(R.layout.activity_main4, null);

        // Obtener referencia al linearLayout (puedes usar el tipo de layout que estés utilizando)
        LinearLayout linearLayout = rootView.findViewById(R.id.linearLayoutContainer);

        // Crear un nuevo TextView dinámicamente
        TextView nuevoTextView = new TextView(this);
        nuevoTextView.setText("Hola, Alberto");

        // Agregar el nuevo TextView al contenedor
        linearLayout.addView(nuevoTextView);

        // Establecer la vista inflada en la actividad
        setContentView(rootView);
    }
}

File: Prog Multimedia/2ºTrimestre/Layouts Inflates/LayoutsInfalyus/app/src/main/java/Inflate1/MainActivity4.kt
package Inflate1

import android.view.View
import com.example.layoutsinfalyus.R
import android.os.Bundle
import android.view.LayoutInflater
import android.widget.LinearLayout
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity4 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Inflar el diseño XML correcto (activity_main.xml)
        val rootView: View = LayoutInflater.from(this).inflate(R.layout.activity_main4, null)

        // Obtener referencia al LinearLayout
        val linearLayout = rootView.findViewById<LinearLayout>(R.id.linearLayoutContainer)

        // Crear un nuevo TextView dinámicamente
        val nuevoTextView = TextView(this)
        nuevoTextView.text = "Hola, Alberto"

        // Agregar el nuevo TextView al contenedor
        linearLayout.addView(nuevoTextView)

        // Establecer la vista inflada en la actividad
        setContentView(rootView)
    }
}
